authzed/api/v1alpha1/lookupwatch_service.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+package authzed.api.v1alpha1;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1alpha1";
+option java_package = "com.authzed.api.v1alpha1";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v1/core.proto";
+
+// LookupWatchService is used to receive a stream of updates for resources of a
+// specific (resource type, permission, subject) combination.
+service LookupWatchService {
+
+  // WatchAccessibleResources initiates a watch for accessibility changes for the provided
+  // (resource type, permission, subject) pair.
+  rpc WatchAccessibleResources(WatchAccessibleResourcesRequest)
```

Comment:
Let's name this RPC `WatchResources`, `WatchResourcesRequest`, `WatchResourcesResponse`

Reply:
Sounds good. I was just following the proposal in https://github.com/authzed/spicedb/issues/207